apps/expo/src/components/workout/frequency_graph.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import {Text, View, Dimensions} from 'react-native'
+
+// Using the new chart library to see how it looks.
+// Properties: https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/HEAD/docs/LineChart/LineChartProps.md
+import {LineChart} from 'react-native-gifted-charts'
+
+// To get the user's data, in particular the WorkoutLogs
+import { api } from '~/utils/trpc/api'
+import RotatingBarbellIcon from '../notif/RotatingBarbellIcon'
+
+
+// Just use react-native-chart-kit to display the user's workout across each month for current year.
+export default function DisplayWorkoutFrequencyGraph({userID} : {userID: number}){
+    let values = [
```

Comment:
is this mock data?

Reply:
The values array is necessary, but I can zero out the rest of the array until it gets populated if that's what you were asking. I can look into finding a way to truncate the graph but not sure if that would mess up sizing.